packages/jest-mock/src/index.ts

```diff
@@ -1192,6 +1192,10 @@ export class ModuleMocker {
     this._mockState = new WeakMap();
   }
 
+  hasAnyMocks(): boolean {
+    return this._mockConfigRegistry.size > 0 || this._mockState.size > 0;
```

Comment:
Just realized this doesn't work because WeakMaps specifically do not have a way of checking if they are filled with any values

Reply:
we probably need some separate state thing which just increments or decrements based on creation/reset of states?

Reply:
yeah I think just a simple array of maybe the function names? My idea is that you could have an after hook like this:

```js

afterEach(() => {
  if (jest.hasAnyMocks()) {
     jest.mockedMethods.map(m => console.error(`You have a mock that hasn't been reset called ${m}`)
  }
})
```